src/app/views.py

```diff
@@ -53,6 +55,44 @@ def about(request):
         'app/about.html',context_dict
         )
 
+def submitNewLicense(request):
+    """ View for submit new licenses
+    returns submit_new_license.html template
+    """
+    if request.method == 'POST':
+        form = LicenseRequestForm(request.POST, auto_id='%s')
+        if form.is_valid():
+            # process the data in form.cleaned_data as required
+            # ...
+            # redirect to a new URL:
+#            return HttpResponseRedirect('/submit_new_license/')
```

Comment:
Getting an error here for indentation. Please fix.

Reply:
Also, as @goneall  suggested don't comment out old code. This made the `if form.is_valid():` block without any statement.